### PR TITLE
Spiderfy fix - add + for 2 or more points at same spot

### PR DESCRIPTION
**Pull request by dpiaseckisoldevelo**

### CODE REVIEW

File: simple_ui/static/js/refugeeApp/directives/servicesMap.js [hunk 1]
```diff
@@ -73,9 +73,14 @@ angular.module('refugeeApp').directive('servicesMap', function(leafletData, $sta
                 let checkOverlappingServices = (map) => {
                     let locs = Object.values(map._layers).map( (m) => {return m._latlng;} );
                     for (let marker in map._layers) {
-                        if (map._layers[marker]._latlng && map._layers[marker].options.service) {
-                            let locsFiltered = locs.filter( (l) => { return l && l.lng !== map._layers[marker]._latlng.lng && l.lat !== map._layers[marker]._latlng.lat; });
-                            let closestLayer = L.GeometryUtil.closest(map, locsFiltered, map._layers[marker]._latlng, true);
+                        let layer = map._layers[marker];
+                        if (layer._latlng && layer.options.service) {
+                            let locsFiltered = locs.filter( (l) => (l) );
```
Comment (awalkowiak) [inline simple_ui/static/js/refugeeApp/directives/servicesMap.js]: What is the purpose of this line? Does it do anything?

Comment (awalkowiak) [inline simple_ui/static/js/refugeeApp/directives/servicesMap.js]: Is this for filtering out null values?

Comment (dpiaseckisoldevelo) [inline simple_ui/static/js/refugeeApp/directives/servicesMap.js]: Yes, to take out undefined values.

**Comment by awalkowiak** [comment]
I checked it localy, seems like now every marker gets plus sign after zoom change (even if it is not overlapping with anything).

**Pull request merged**